tests/accessor/generic_accessor_linearization_core.cpp

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for generic accessor linearization with generic types
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !SYCL_CTS_COMPILING_WITH_HIPSYCL && !SYCL_CTS_COMPILING_WITH_COMPUTECPP && \
+    !SYCL_CTS_COMPILING_WITH_DPCPP
```

Comment:
I'm confused as to why none of the impls are able to pass this?

Reply:
This test is disabled for hipSYCL because handler::host_task and marray is not implemented yet,
for DPCPP because target::host_task is not implemented,
ComputeCpp because possibility of a SYCL kernel with an unnamed type is not implemented

Reply:
I am worried about these tests we might forget about.
How can we remember to re-enable them later?

Reply:
Aside from adding more FIXME comments in code, may be there is could be some issue created with all disabled tests?

Reply:
You could also create a `TODO_TEST_CASE` (see `macros.h`) as a stand-in.

Reply:
From my understanding TODO_TEST_CASE is used when test in question is not yet implemented. But here is reverse situation, since test is already here but it's disabled because feature is not fully implemented. That's why DISABLED_FOR_TEST_CASE is used in line 23. Maybe tag similat to [todo] could be added to DISABLED_FOR_TEST_CASE macros to track all disabled tests and remember to re-enable them later?

Reply:
Apologies, I did not look at the rest of the diff and thought the entire file was disabled through this preprocessor block. In this case I don't think there's much risk of forgetting to enable the test case at some point, because any implementers seeking conformance will get an error message for this test case when running the CTS against their implementation.

Reply:
But why do we care about skipping this test? At the end if an implementation does not pass the CTS, this is some useful information for the developer.

Reply:
Test case is disabled because these are compile errors and CI for PR would fail.
And "include ..." is skipped because header uses these not yet implemented features.